rsp/runtimes/bundles/org.jboss.tools.rsp.server.tomcat/src/main/resources/servers.json

```diff
@@ -74,6 +74,12 @@
 						"pollerProperties": {
 							"url": "http://localhost:8080"
 						}
+					},
+					"actions": {
+						"showinbrowser": {},
+						"editserverconfiguration": {
+							"path": "conf/server.xml"
```

Comment:
We might consider making the field "paths" instead, comma-separated. If only 1 exists, edit that one. If more than 1 is listed and they exist, present options to user.